.github/workflows/_upload_cesium_token_txt.yml

```
@@ -32,4 +33,18 @@ jobs:
             echo "Token length is invalid. TokenLength: ${token_length}"
             exit 1
           fi
-          gsutil cp ${cesium_ion_token_txt_path} ${{ inputs.gcs_dst }}/cesium_ion_token.txt
\ No newline at end of file
+          echo "token=$(<${cesium_ion_token_txt_path})" >> $GITHUB_OUTPUT
+      - name: Update Secret Manager secret
+        run: |
+          echo -n "${{ steps.ion_token.outputs.token }}" | gcloud secrets versions add ${{ secrets.secret_name }} \
+            --data-file=-
+      - name: Update Cloud Run
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:28: Double quote to prevent globbing and word splitting

(shellcheck)